### PR TITLE
Sample spring postgres graphql

### PR DESCRIPTION
**Pull request by Rentox98**
Signed-off-by: Alberto Cao <albertocao98@gmail.com>

# Pull Request Template

## Description

Creating sample java application using postgres and graphql, tested using Keploy.

Fixes [keploy/keploy#911](https://github.com/keploy/keploy/issues/911)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

The demo has been tested locally on a WSL2 environment with Docker, following the steps outlined in the project's README.

## Additional Context (Please include any Screenshots/gifs if relevant)

Capturing requests using Keploy record.

doc_1

From the GraphQL GUI run the queries

doc_3

Record requests with Keploy
doc_2

Test with Keploy
doc_4
doc_5

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings
- [x] I have tagged the reviewers in a comment below incase my pull request is ready for a review
- [x] I have signed the commit message to agree to Developer Certificate of Origin (DCO) (to certify that you wrote or otherwise have the right to submit your contribution to the project.) by adding "--signoff" to my git commit command.

### CODE REVIEW

**Comment by Rentox98** [comment]
@Sonichigo Hi, I tagged you for the review of my pull request, if you have any doubts ask me please,
Thanks

**Comment by nehagup** [comment]
@Rentox98 thanks for the PR, we'll review this soon. :)

File: spring-boot-postgres-graphql/README.md [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+- Depending on your OS and preference (Docker/Native), you can set up Keploy using the one-click installation method:
+   ```bash
+   curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+   ```
+
+## Setup DB
+
+You have two possible cases:
+
+### Use your postgres installation
```
Comment (Sonichigo) [inline spring-boot-postgres-graphql/README.md:47]: There is the problem with the postresql start section here.In record mode, postgres is not connecting with application. While trying prostgres start cmd , better to remove it as with docker image postgres it works fine.

**Review by Sonichigo — CHANGES_REQUESTED** [review]

**Comment by Sonichigo** [comment]
Rest LGTM and it works! Great contribution @Rentox98

File: spring-boot-postgres-graphql/README.md (hunk 1 above)
Comment (Rentox98) [reply spring-boot-postgres-graphql/README.md]: Thanks for the review, I corrected that part.

**Review by Sonichigo — APPROVED** [review]

**Pull request merged**